core/modules/dblog/dblog.module

```diff
@@ -151,19 +151,19 @@ function dblog_watchdog(array $log_entry) {
  * Implements hook_form_FORM_ID_alter() for system_logging_settings().
  */
 function dblog_form_system_logging_settings_alter(&$form, $form_state) {
+  require_once(BACKDROP_ROOT . '/core/includes/date.inc');
```

Comment:
Same as above: core/includes/date.inc should already be included in function _backdrop_bootstrap_full().